python/route_guide/run_client.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# This is where you have cloned out the https://github.com/grpc/grpc repository
+# And built gRPC Python.
+# ADJUST THIS PATH TO WHERE YOUR ACTUAL LOCATION IS
+GRPC_ROOT=~/github/grpc
+
+$GRPC_ROOT/python2.7_virtual_environment/bin/python route_guide_client.py
```

Comment:
Add "-B " before "route_guide_client.py" to suppress the generation of .pyc files.